examples/mugen/test/retrieval/test_video_clip.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import pytest
+import torch
+from examples.mugen.retrieval.video_clip import TextEncoder
+
+from test.test_utils import assert_expected, set_rng_seed
+
+
+class TestTextEncoder:
```

Comment:
For simplicity we can write this as a function since it's just one test.

Reply:
I added a test for the attention mask, but if there was still only one test -- I prefer to group the text encoder test in its own class because this file will later also contain tests for VideoEncoder and the model builder, and it's a nice way to organize the text-specific fixtures and potential future text encoder tests. It also follows the pattern set by other test files in our repo (although having test classes might just be a unittest requirement).